shell/browser/net/proxying_url_loader_factory.h

```diff
@@ -238,6 +238,7 @@ class ProxyingURLLoaderFactory
   content::BrowserContext* const browser_context_;
   const int render_process_id_;
   uint64_t* request_id_generator_;  // managed by ElectronBrowserClient
+  std::set<int32_t> intercepted_requests_;
```

Comment:
I think this set grows without bound? Which would be a memory leak and thus not acceptable to merge.